src/api/endpoints/logged_activities/secretary_review.py

```diff
@@ -53,20 +53,26 @@ def put(self, logged_activity_id):
 
         logged_activity.status = payload.get('status')
         if logged_activity.status == "pending":
+            # Send approved notification to success-ops
             SlackNotification.send_notification(self, roles, users, message)
 
+            # Send approved notification to the respective fellow
+            user_email = logged_activity.user.email
+
+            message = f"Your logged society points worth {logged_activity.value} described as " + \
```

Comment:
Can we instead do something like 

`f"APPROVED. Your activity points for {logged_activity.description} logged on { logged_activity.date} have been approved by your Society's Secretary. " `

I realize you might have to add the same to the other notifications. Let me know how much effort it would take altogether.

Reply:
Let me work on this first, then maybe we can have the others done as bug stories.